.github/workflows/booster-schedule.yaml

```diff
@@ -0,0 +1,56 @@
+name: Boost backers' rewards in an active NFT campaign
+
+permissions:
+  contents: write
+
+env:
+  BOOST: '30'
+  NFT_CONTRACT: '0x1EEA464C8632A71A2E9B364725da6e09aaed2Ddc'
+  ENVIRONMENT: 'mainnet'
+  BOOST_DATA_BRANCH: 'boost_data/v1.7.0'
```

Comment:
This should be `boost_data/mainnet`

Reply:
or it could be built using the env variable

Reply:
because the `ENVIRONMENT` field is declared as an env variable just like `BOOST_DATA_BRANCH`, Github Actions wont resolve the `ENVIRONMENT` variable into the other one.